lightkurve/search.py

```diff
@@ -240,7 +232,8 @@ def _default_download_dir(self):
         return download_dir
 
 
-def search_targetpixelfile(target, radius=None, cadence='long', quarter=None,
+def search_targetpixelfile(target, radius=None, cadence='long',
+                           project=['Kepler', 'K2', 'TESS'], quarter=None,
```

Comment:
Let's make sure we add a good docstring for `project` here and in `search_lightcurvefile`!